examples/forecasts/advanced_forecasting_example.py

```diff
@@ -0,0 +1,57 @@
+from autogluon.forecasting import ForecastingPredictor
```

Comment:
Can you rename the directory `examples/forecasts/` to `examples/forecasting` to match the task name?

Reply:
> Can you rename the directory `examples/forecasts/` to `examples/forecasting` to match the task name?

Done.